spec/controllers/teams_controller/destroy_spec.rb

```diff
@@ -62,12 +62,8 @@
           delete :destroy, params: params
         end
 
-        it 'displays a flash alert' do
-          expect(flash[:alert]).to eq "This Directorate still has active business units. Please deactivate all business units of the Directorate before deactivating the Directorate"
```

Comment:
Doesn't `TeamsController#deletion_validation` set the flash message "Unable to delete team: team still has active children"? We should expect to see that.

Reply:
Deactivation of Business Groups and Directorates is not implements and not part of this ticket.  A lot of the work has been done, but needs to be refactored so that checking the validity of deleting the record is an ActiveRecord validation, and the error message is displayed as a validation error, not a flash message.  This can all be done and tested in the ticket to deactivate business groups and directorates.

Reply:
Ok. Feels like we should protect against deactivation of Business Groups or Directorates in the policy if we don't want to allow it, not just rely on the button not showing, which is what it looks like now.